src/legacy/ui/ui_bundles/app_entry_template.js

```diff
@@ -26,7 +26,13 @@ export const appEntryTemplate = (bundle) => `
  * context: ${bundle.getContext()}
  */
 
-// import global polyfills before everything else
+// ensure the csp nonce is set in the dll
+import 'dll/set_csp_nonce';
+
+// set the csp nonce in the primary webpack bundle too
+__webpack_nonce__ = window.__webpack_nonce__;
```

Comment:
Note for followup work: I don't think `window.__webpack_nonce__` is really being picked up by webpack at all, so we probably should rename it to `__kbn_nonce__` or something to make this distinction more explicit.